core/core-dotspacemacs.el

```diff
@@ -171,7 +171,7 @@ a layer lazily."
 wrapped in a layer. If you need some configuration for these
 packages then consider to create a layer, you can also put the
 configuration in `dotspacemacs/user-config'."
-  '(repeat symbol)
+  '(repeat (choice symbol (cons symbol sexp)))
```

Comment:
maybe the type can be stricter here.

In the case:

```
   dotspacemacs-additional-packages '(
     amx
     (beacon :location (recipe :fetcher github :repo "Malabarba/beacon")))
```
it looks like in the cons case we always have first 2 elements as symbols.  So maybe '(repeat (choice symbol (cons symbol (const symbol sexp))) will work :thinking:

Reply:
On the other hands it's too cryptic.